### PR TITLE
Adds space prefab: Shootin' Range

### PR DESCRIPTION
**Pull request by nefarious6th**
[mapping]
## About the PR 
Adds another space prefab, a shooting range that has some experimental energy weapons and foam dart guns.

## Why's this needed?
Prefabs are fun. Thought this would be a fun hang-out spot for people to bring their foam dart guns if they didn't want to run around and bug people on station.

### CODE REVIEW

**Review by flrsh — APPROVED** [review]
Cool prefab!

**Review by Tarmunora — APPROVED** [review]

File: code/obj/decorations.dm [hunk 1]
```diff
@@ -1005,3 +1005,81 @@ obj/decoration/ceilingfan
...
+
+/obj/item/gun/laser_pistol/prototype
+	name = "prototype laser pistol"
+	icon = 'icons/obj/decoration.dmi'
+	desc = "You've never heard of this pistol before...who made it?"
+	icon_state = "e_laser_pistol"
+
+//stolen code for anchorable and movable target sheets. cannot get projectile tracking on them to work right now so. oh well. help appreciated!
+/obj/item/caution/target_sheet
+	desc = "A paper sheet of a figure on a cardboard backing."
```
Comment (virvatuli-gh) [inline code/obj/decorations.dm:1038]:
```suggestion
	desc = "A paper silhouette target sheet with a cardboard backing."
```
technical terms for the nerds

File: code/obj/item/paper.dm [hunk 2]
```diff
@@ -1599,3 +1599,16 @@ exposed to overconfident outbursts on the part of individuals unqualifed to embo
 	desc = "Somebody took whatever was in here."
 	icon_state = "postit-writing"
 	info = {"<h2>IOU</h2>"}
+
+/obj/item/paper/shooting_range_note1 //shooting range prefab junk
+	name = "secure safe note"
+	desc = "Someone left a reminder in neat cursive. The post-it looks nearly new."
+	icon_state = "postit-writing"
+	info = {"*Experimental ray gun - DO NOT FIRE IN A CLOSED SPACE. Waiting for Olwen to fix...whenever she's back...<br><b><u>*Dinner date is on <s>Tuesday</s>  <s>Fri.</s></s><br>
```
Comment (virvatuli-gh) [inline code/obj/item/paper.dm:1607]:
```suggestion
	info = {"*Experimental ray gun - DO NOT FIRE IN A CLOSED SPACE. Waiting for Olwen to fix... whenever she's back...<br><b><u>*Dinner date is on <s>Tuesday</s>  <s>Fri.</s></s><br>
```
minor grammar

File: code/obj/decorations.dm [hunk 3]
```diff
@@ -1005,3 +1005,81 @@ obj/decoration/ceilingfan
...
+		if ((user.r_hand == src || user.l_hand == src) && src.contents && length(src.contents))
+			user.visible_message("The cell on this is corroded. Good luck getting this thing to fire ever again!")
+			src.add_fingerprint(user)
+		else
+			return ..()
+
+/obj/item/gun/laser_pistol/prototype
+	name = "prototype laser pistol"
+	icon = 'icons/obj/decoration.dmi'
+	desc = "You've never heard of this pistol before...who made it?"
```
Comment (virvatuli-gh) [inline code/obj/decorations.dm:1033]:
```suggestion
	desc = "You've never heard of this pistol before... who made it?"
```
minor grammar

**Review by virvatuli-gh — COMMENTED** [review]
Super awesome and I'm glad to see that gun sprite you showed me eons ago finally making an appearance! Just a few minor things I noticed while testing, I love the prefab

File: code/obj/decorations.dm (hunk 3 above)
Comment (nefarious6th) [reply code/obj/decorations.dm]: fixed up all the grammar and spelling things! ty!

**Review by virvatuli-gh — APPROVED** [review]

**Pull request merged**